jlink-jib/src/main/kotlin/com/ryandens/jlink/jib/JlinkJibPlugin.kt

```diff
@@ -56,6 +56,10 @@ class JlinkJibPlugin : JibGradlePluginExtension<Void>, Plugin<Project> {
     }
 
     override fun apply(project: Project) {
+        checkNotNull(project.extensions.findByName("jib")) {
```

Comment:
rather than finding the extenison, use the `pluginManager` to detect the presence of the jib plugin